.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,18 @@
+name: CI_pull_request
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@slalpha5
+        with:
+          args:
+            build -c --skip-tests
```

Comment:
Why are we skipping tests? Its better to run tests after PR is merged right?

Reply:
After PR merged run will be considered as on push job. Here it's only for creating a PR.